hack/.cleanupUtils

```diff
@@ -0,0 +1,21 @@
+# shellcheck shell=bash
+
+cleanClusters() {
+	# Delete existing kind clusters
+	clusterCount=$(${KIND_BIN} get clusters | grep ${KIND_CLUSTER_PREFIX} | wc -l)
+	if ! [[ $clusterCount =~ "0" ]] ; then
+		echo "Deleting previous kind clusters."
+		${KIND_BIN} get clusters | grep ${KIND_CLUSTER_PREFIX} | xargs ${KIND_BIN} delete clusters
+	fi	
+}
+
+cleanNetwork() {
+    # Delete the network
+    echo "Deleting mctc network"
+    docker network rm mctc || true
+}
+
+cleanup() {
+	cleanClusters
+	cleanNetwork
+}
```

Comment:
Can this cleanup those local proxies for the dsashboards as well?

Reply:
I added the ability to stop the proxies in #31. We can get this merged first and then I update that PR to add a call to the function in the cleanup